frontend/packages/multicluster-sdk/README.md

```diff
@@ -169,6 +169,62 @@ Array with `isAllowed` and `loading` values.
 
 [:link: Source](https://github.com/stolostron/console/blob/main/frontend/packages/multicluster-sdk/tree/../src/api/useFleetPrometheusPoll/index.ts#L13)
 
+### :gear: useFleetSearchPoll
+
+A React hook that provides fleet-wide search functionality using ACM's Console's search API.
+
+| Function | Type |
+| ---------- | ---------- |
+| `useFleetSearchPoll` | `UseFleetSearchPoll` |
+
+Parameters:
+
+* `watchOptions`: - Configuration options for the resource watch
+* `watchOptions.groupVersionKind`: - The group, version, and kind of the resource to search for
+* `watchOptions.limit`: - Maximum number of results to return (defaults to -1 for no limit)
+* `watchOptions.namespace`: - Namespace to search in (only used if namespaced is true)
+* `watchOptions.namespaced`: - Whether the resource is namespaced
+* `watchOptions.name`: - Specific resource name to search for (exact match)
+* `watchOptions.isList`: - Whether to return results as a list or single item
+* `advancedSearch`: - Optional array of additional search filters
+* `advancedSearch[].property`: - The property name to filter on
+* `advancedSearch[].values`: - Array of values to match for the property
+* `pollInterval`: - Optional polling interval in milliseconds. If provided, the search will be repeated at this interval
```

Comment:
Can we use seconds as the unit? Because the shortest polling interval allowed should be 30 seconds, so a millisecond resolution is not needed. I also want to make polling the default since it's part of the name.
I propose:

- pollInterval not specified: polls every 30 seconds
- pollInterval 0-30 inclusive: polls every 30 seconds
- pollInterval >30: polls at the given interval
- pollInterval false or negative: polling disabled